components/suggestion/at_mention_provider/at_mention_provider.jsx

```diff
@@ -24,13 +29,15 @@ export default class AtMentionProvider extends Provider {
         this.lastCompletedWord = '';
         this.lastPrefixWithNoResults = '';
         this.triggerCharacter = '@';
+        this.getProfilesInChannel = makeGetProfilesInChannel();
+        this.addLastViewAtToProfiles = makeAddLastViewAtToProfiles();
     }
 
     // setProps gives the provider additional context for matching pretexts. Ideally this would
     // just be something akin to a connected component with access to the store itself.
-    setProps({currentUserId, profilesInChannel, autocompleteUsersInChannel, useChannelMentions, autocompleteGroups, searchAssociatedGroupsForReference, priorityProfiles}) {
+    setProps({currentUserId, currentChannelId, autocompleteUsersInChannel, useChannelMentions, autocompleteGroups, searchAssociatedGroupsForReference, priorityProfiles}) {
         this.currentUserId = currentUserId;
-        this.profilesInChannel = profilesInChannel;
+        this.currentChannelId = currentChannelId;
```

Comment:
Just the one thing I was nitpicking about before, this should be called `channelId` instead of `currentChannelId`

Reply:
ahh i forgot that